bindings/java/src/main/java/org/ucx/jucx/ucp/UcpRemoteMemory.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+package org.ucx.jucx.ucp;
+
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+
+/**
+ * Class that represents remote memory with it's address and remote key buffer.
+ */
+public class UcpRemoteMemory implements Serializable {
+
+    private ByteBuffer remoteKey;
+
+    private long address;
+
+    public ByteBuffer getRemoteKey() {
+        return remoteKey;
+    }
+
+    public long getAddress() {
+        return address;
+    }
+
+    public UcpRemoteMemory(ByteBuffer remoteKey, long address) {
+        this.address = address;
+        this.remoteKey = remoteKey;
+    }
+
+    public UcpRemoteMemory(UcpMemory memory) {
+        this.address = memory.getAddress();
+        this.remoteKey = memory.getRemoteKeyBuffer();
```

Comment:
remote key is not the same as remote key buffer.. need to unpack it. where does unpack happen?

Reply:
Here: https://github.com/openucx/ucx/pull/3536/files#diff-493a0aa822b2de2016503b5fdd672feeR93

Reply:
unpack could be expensive operation, need to do it when creating remote memory object, and not for every get operation